File: SpecialFunctions.py
import pandas as pd
def formatDataset(data):
    df = pd.DataFrame(data)
    df['datetime'] = pd.to_datetime(df['date'])
    df = df.set_index('datetime')
    df = df.drop('date', axis=1)
    return df
def formatDataset1(df):
    columns_to_convert = ['open', 'high', 'low', 'close', 'volume']
    for column in columns_to_convert:
        df[column] = df[column].astype(float)
    return df

File: specialFunctions.jl
module specialFunctions

export formatDataset3, formatDataset2

using DataFrames
using Dates

function formatDataset(data)
    df = DataFrame(data)
    df.datetime = DateTime.(df.date)
    select!(df, Not(:date))
    return df
end

function formatDataset1(df)
    columns_to_convert = [:open, :high, :low, :close, :volume]
    df[!, columns_to_convert] .= float.(df[!, columns_to_convert])
    return df
end






end